src/components/Swap/Toolbar/index.tsx

```diff
@@ -61,7 +61,7 @@ function CaptionRow() {
       default:
     }
 
-    if (state === TradeState.LOADING) {
+    if (state === TradeState.LOADING && trade == null) {
```

Comment:
hmm is it intentional to not show the loading state on subsequent fetches? i think as a user i'd expect there to be a loading state if i changed my input amount or currency, for example 🤔

Reply:
see the PR description or deploy preview - we do have a loading state, newly added to the `Trade` component below